javascript/src/main/pages/Admin/SearchedTerms.js

```diff
@@ -30,4 +30,4 @@ const SearchedTerms = () => {
         </>
```

Comment:
also need to make this change in SearchedTerms.js:
![image](https://user-images.githubusercontent.com/35274879/110881721-ac74b600-8295-11eb-92c6-ec0e250e21c0.png)
otherwise it's just redundant. you should just be able to import the new component file you made and get rid of all the abstracted code from this file.